internal/handlers/pg/msg_getlog.go

```diff
@@ -83,27 +83,34 @@ func (h *Handler) MsgGetLog(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		))
 
 	case "startupWarnings":
-		pv, _, _ := strings.Cut(h.StateProvider.Get().HandlerVersion, " ")
-		mv := version.Get()
+		state := h.StateProvider.Get()
+
+		info := version.Get()
+
+		hv, _, _ := strings.Cut(state.HandlerVersion, " ")
+		if hv != "" {
+			hv = " " + hv
+		}
 
 		startupWarnings := []string{
-			"Powered by FerretDB " + mv.Version + " and PostgreSQL " + pv + ".",
+			"Powered by FerretDB " + info.Version + " and PostgreSQL" + hv + ".",
 			"Please star us on GitHub: https://github.com/FerretDB/FerretDB.",
 		}
 
-		state := h.StateProvider.Get()
-
 		switch {
 		case state.Telemetry == nil:
 			startupWarnings = append(
 				startupWarnings,
-				"The telemetry state is undecided; the first report will be sent soon. "+
```

Comment:
We should not say "the first report will be sent soon" after the first report is sent.